src/Take.Blip.Builder/Actions/ExecuteScript/ExecuteScriptAction.cs

```diff
@@ -59,5 +78,19 @@ public override async Task ExecuteAsync(IContext context, ExecuteScriptSettings
                 await context.DeleteVariableAsync(settings.OutputVariable, cancellationToken);
             }
         }
+
+        private void CheckMemoryUsage(IContext context, DebugInformation debugInformation)
+        {
+            if (debugInformation.CurrentMemoryUsage >= _configuration.ExecuteScriptLimitMemoryWarning)
+            {
+                var currentActionTrace = context.GetCurrentActionTrace();
+
+                if (currentActionTrace != null)
+                {
+                    currentActionTrace.Warning =
+                        $"The script memory usage is above the warning threshold of {_configuration.ExecuteScriptLimitMemoryWarning} bytes({debugInformation.CurrentMemoryUsage})";
```

Comment:
It is not clear what the number in the parentheses represents.
```suggestion
                        $"The script memory usage ({debugInformation.CurrentMemoryUsage} bytes) is above the warning threshold of {_configuration.ExecuteScriptLimitMemoryWarning} bytes";
```